### PR TITLE
Some javadoc and param names in mostly server related classes.

### PR DESCRIPTION
**Pull request by i509VCB**

### CODE REVIEW

File: mappings/net/minecraft/client/gui/hud/InGameHud.mapping [hunk 1]
```diff
@@ -72,6 +72,11 @@ CLASS net/minecraft/class_329 net/minecraft/client/gui/hud/InGameHud
 	METHOD method_1761 renderPumpkinOverlay ()V
 	METHOD method_1762 renderHotbarItem (IIFLnet/minecraft/class_1657;Lnet/minecraft/class_1799;)V
 	METHOD method_1763 setTitles (Ljava/lang/String;Ljava/lang/String;III)V
+		ARG 1 title
+		ARG 2 subtitle
+		ARG 3 titleFadeTicks
```
Comment (YanisBft) [inline mappings/net/minecraft/client/gui/hud/InGameHud.mapping]:
```suggestion
		ARG 3 titleFadeInTicks
```

File: mappings/net/minecraft/server/rcon/RconBase.mapping [hunk 2]
```diff
@@ -3,9 +3,15 @@ CLASS net/minecraft/class_3359 net/minecraft/server/rcon/RconBase
 	FIELD field_14424 description Ljava/lang/String;
 	FIELD field_14425 server Lnet/minecraft/class_2994;
 	FIELD field_14426 sockets Ljava/util/List;
+	FIELD field_14428 THREAD_COUNTER Ljava/util/concurrent/atomic/AtomicInteger;
 	FIELD field_14429 serverSockets Ljava/util/List;
+	FIELD field_14430 RCON_LOGGER Lorg/apache/logging/log4j/Logger;
```
Comment (YanisBft) [inline mappings/net/minecraft/server/rcon/RconBase.mapping]:
```suggestion
	FIELD field_14430 LOGGER Lorg/apache/logging/log4j/Logger;
```
It's a static field, only LOGGER should suffice

Comment (i509VCB) [reply mappings/net/minecraft/server/rcon/RconBase.mapping]: When I tried `LOGGER`, it said the name of the field would conflict with the logger in `RconClient`

Comment (YanisBft) [reply mappings/net/minecraft/server/rcon/RconBase.mapping]: It's a known Enigma bug, the only solution right now is to edit the name in the mapping file directly. Static fields or methods don't conflict with parent/child classes

Comment (i509VCB) [reply mappings/net/minecraft/server/rcon/RconBase.mapping]: oh okay

**Review by YanisBft — APPROVED** [review]

File: mappings/net/minecraft/client/gui/hud/InGameHud.mapping [hunk 3]
```diff
@@ -72,6 +72,11 @@ CLASS net/minecraft/class_329 net/minecraft/client/gui/hud/InGameHud
 	METHOD method_1761 renderPumpkinOverlay ()V
 	METHOD method_1762 renderHotbarItem (IIFLnet/minecraft/class_1657;Lnet/minecraft/class_1799;)V
 	METHOD method_1763 setTitles (Ljava/lang/String;Ljava/lang/String;III)V
+		ARG 1 title
+		ARG 2 subtitle
+		ARG 3 titleFadeInTicks
+		ARG 4 titleRemainTicks
+		ARG 5 titleFadeOutTicks
```
Comment (liach) [inline mappings/net/minecraft/client/gui/hud/InGameHud.mapping]: don't need title prefix in the last 3 params

File: mappings/net/minecraft/server/LanServerPinger.mapping [hunk 4]
```diff
@@ -1,4 +1,7 @@
 CLASS net/minecraft/class_1133 net/minecraft/server/LanServerPinger
+	COMMENT Used to send UDP multicasts, which notify other clients of a local game on the same network.
+	COMMENT
+	COMMENT <p>These multicasts will always be sent to {@code 224.0.2.60:4445} where other clients can listen for local games.
```
Comment (liach) [inline mappings/net/minecraft/server/LanServerPinger.mapping:4]: new paragraph?

File: mappings/net/minecraft/server/LanServerPinger.mapping [hunk 5]
```diff
@@ -9,8 +12,24 @@ CLASS net/minecraft/class_1133 net/minecraft/server/LanServerPinger
 		ARG 1 motd
 		ARG 2 addressPort
 	METHOD method_4818 createAnnouncement (Ljava/lang/String;Ljava/lang/String;)Ljava/lang/String;
+		COMMENT Creates a server announcement.
+		COMMENT
+		COMMENT <p>Server announcements are made of the following:
+		COMMENT <ul>
+		COMMENT <li>[MOTD] // Specifies the beginning of the message of the day
```
Comment (liach) [inline mappings/net/minecraft/server/LanServerPinger.mapping]: consider using blockquote pre tags and unordered/ordered lists

**Review by YanisBft — APPROVED** [review]

File: mappings/net/minecraft/client/gui/hud/InGameHud.mapping [hunk 6]
```diff
@@ -74,9 +74,9 @@ CLASS net/minecraft/class_329 net/minecraft/client/gui/hud/InGameHud
 	METHOD method_1763 setTitles (Ljava/lang/String;Ljava/lang/String;III)V
 		ARG 1 title
 		ARG 2 subtitle
-		ARG 3 titleFadeInTicks
-		ARG 4 titleRemainTicks
-		ARG 5 titleFadeOutTicks
+		ARG 3 titleFadeIn
+		ARG 4 titleRemain
+		ARG 5 titleFadeOut
```
Comment (YanisBft) [inline mappings/net/minecraft/client/gui/hud/InGameHud.mapping]:
```suggestion
		ARG 3 fadeInTicks
		ARG 4 remainTicks
		ARG 5 fadeOutTicks
```
You removed the suffix, not the prefix

File: mappings/net/minecraft/server/rcon/RconBase.mapping [hunk 7]
```diff
@@ -1,7 +1,31 @@
...
+	FIELD field_14428 THREAD_COUNTER Ljava/util/concurrent/atomic/AtomicInteger;
+	FIELD field_14429 serverSockets Ljava/util/List;
+	FIELD field_14430 LOGGER Lorg/apache/logging/log4j/Logger;
 	FIELD field_14431 running Z
+	METHOD <init> (Lnet/minecraft/class_2994;Ljava/lang/String;)V
+		ARG 1 server
+		ARG 2 description
+	METHOD method_14721 closeSocket (Ljava/net/ServerSocket;)Z
+		ARG 1 socket
+	METHOD method_14722 log (Ljava/lang/String;)V
```
Comment (liach) [inline mappings/net/minecraft/server/rcon/RconBase.mapping:15]: How does this differentiate from other logging methods?

**Review by liach — COMMENTED** [review]
Also please take yanis' suggestion.

**Comment by liach** [comment]
@i509VCB mind target this pr against 17a?

File: mappings/net/minecraft/command/arguments/ArgumentTypes.mapping [hunk 8]
```diff
@@ -17,3 +17,7 @@ CLASS net/minecraft/class_2316 net/minecraft/command/arguments/ArgumentTypes
 		FIELD field_10924 argClass Ljava/lang/Class;
 		FIELD field_10925 id Lnet/minecraft/class_2960;
 		FIELD field_10926 serializer Lnet/minecraft/class_2314;
+		METHOD <init> (Ljava/lang/Class;Lnet/minecraft/class_2314;Lnet/minecraft/class_2960;)V
+			ARG 1 argClass
```
Comment (Runemoro) [inline mappings/net/minecraft/command/arguments/ArgumentTypes.mapping]: You use `argumentName` elsewhere, so this should be `argumentClass` to be consistent.

File: mappings/net/minecraft/server/rcon/RconBase.mapping (hunk 7 above)
Comment (i509VCB) [reply mappings/net/minecraft/server/rcon/RconBase.mapping:15]: Logs a message to the dedicated server hosting the rcon connection

**Review by liach — APPROVED** [review]

**Review by Runemoro — APPROVED** [review]

**Review by Juuxel — APPROVED** [review]

**Pull request merged**